pkgs/development/compilers/gerbil/ftw.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, gerbilPackages, ... }:
+
+{
+  pname = "ftw";
+  version = "unstable-2022-01-14";
+  git-version = "8ba16b3";
+  softwareName = "FTW: For The Web!";
+  gerbil-package = "drewc/smug";
```

Comment:
Stupid question: elsewhere pname and gerbil-package are related but not here, what is going on?

Reply:
You found a bug indeed. Probably a bad copy-paste, for a field that is not automatically updated by my update script.

Reply:
(It so happens that this bug has no impact on the nix packaging, because there is no version file in this gerbil package.)